### PR TITLE
Lon 1376

### PR DESCRIPTION
**Pull request by vguzmanp**

### CODE REVIEW

File: Tests/Services/TestPlayEditorState.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+			data.project = new LMProjectVM {
+				Model = new LMProject ()
+			};
+			data.settings = new PlayEventEditionSettings ();
+
+			// This should never happen in the real application:
+			data.play = new TimelineEventVM { Model = new TimelineEvent { Name = "Event1" } };
+
+			// Act & Assert
+			Assert.ThrowsAsync<InvalidCastException> (async () => await state.LoadState (data));
```
Comment (ylatuya) [inline Tests/Services/TestPlayEditorState.cs]: Not sure why you are testing that. How is this test helping to detect any error?  It's something that will always happen, if you pass incorrect values in data you will have an exception.

File: Tests/Services/TestPlayEditorState.cs [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+		}
+
+		[Test]
+		public async Task ShowState_ControllersStarted ()
+		{
```
Comment (ylatuya) [inline Tests/Services/TestPlayEditorState.cs]: This isn't a test for this state, it's a test for the base class.

File: Tests/Services/TestPlayEditorState.cs [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+using VAS.Core.Interfaces.GUI;
+using VAS.Core.Interfaces.MVVMC;
+using VAS.Core.MVVMC;
+using VAS.Core.Store;
+using VAS.Core.ViewModel;
+
+namespace Tests.Services
+{
+	[TestFixture]
+	public class TestPlayEditorState
```
Comment (ylatuya) [inline Tests/Services/TestPlayEditorState.cs]: Now that you are creating a test for a state, I would start doing it in a more BDD fashion as we discussed in the planning.

Comment (vguzmanp) [reply Tests/Services/TestPlayEditorState.cs]: @ylatuya Is this what you meant? https://github.com/LongoMatch/longomatch-core/pull/375/commits/c626573fc1ee79df26f36a51d9ed4d2389c5ebee

Comment (ylatuya) [reply Tests/Services/TestPlayEditorState.cs]: yes

**Pull request merged**